contextualized/networks/trainers.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+from contextualized.regression.trainers import RegressionTrainer
+
+    
+class CorrelationTrainer(RegressionTrainer):
```

Comment:
Is correlation a network-specific function? Should we have a CorrelationTrainer in contextualized.regression.trainers?